app/v2/notifications/post_notifications.py

```diff
@@ -84,6 +84,13 @@
 )
 
 
+# TODO: replace with the real thing
+class RedisQueue:
+    @staticmethod
+    def publish(notification):
```

Comment:
The latest signature is:
```
def publish(self, serializable: Serializable):
```

The latest is there:
https://github.com/cds-snc/notification-api/blob/d97138603a66ce251185516c53b5a59b4ffc196a/app/queue.py#L212